.github/workflows/generalized-deployments-reproducible.yaml

```diff
@@ -13,6 +14,8 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+      with:
```

Comment:
I don't think we need this. We dealt with the issue here:
https://github.com/brave-intl/general-docker-build-pipeline/commit/15a294c6675dc3a1b2c06d5fc4a4e173e897d87f

Reply:
Removed in ae958958c287b57488983cec14a3e9927060d307.